### PR TITLE
Adds massa web3

### PR DESCRIPTION
**Pull request by qdrn**

### CODE REVIEW

**Comment by qdrn** [comment]
I've also added @Thykof and @Trottoir to have a dev perspective.

File: web3-dev/massa-web3.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Documentation for the massa-web3 library is available on the
+`Github repository <https://github.com/massalabs/massa-web3>`_. The library is published
+on npmjs under `@massalabs/massa-web3 <https://www.npmjs.com/package/@massalabs/massa-web3>`_
+
+
+`create-react-app-massa <https://github.com/massalabs/create-react-app-massa>`_ is a
+minimal React template made for our massa-web3 library.
+
+You'll find several examples of frontends using the `massa-web3` library
+in the `massa-sc-examples <https://github.com/massalabs/massa-web3>`_ repository:
```
Comment (Thykof) [inline web3-dev/massa-web3.rst:23]: do you mean this one https://github.com/massalabs/massa-sc-examples?

File: web3-dev/useful-resources.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Useful resources
+################
+
+Here is a list of useful resources for smart contract development.
+
+- `massa-web3 <https://github.com/massalabs/massa-web3>`_ is a TypeScript library
+  that allow you to interact with the Massa blockchain through a local or remote Massa node.
+- `create-react-app-massa <https://github.com/massalabs/create-react-app-massa>`_ is a minimal
+  React template made for our massa-web3 library.
+- `massa-sc-examples <https://github.com/massalabs/massa-web3>`_ is a collection of
```
Comment (Thykof) [inline web3-dev/useful-resources.rst:15]: same here do you mean https://github.com/massalabs/massa-sc-examples/tree/main/blog?

**Review by Thykof — CHANGES_REQUESTED** [review]

File: web3-dev/massa-web3.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- `blog <https://github.com/massalabs/massa-sc-examples/tree/main/blog>`_ is an
+  example of a decentralized blog platform.
+- `tic-tac-toe <https://github.com/massalabs/massa-sc-examples/tree/main/games/game-of-life>`_
```
[the remark this answers is not in the document]
Comment (Thykof) [reply web3-dev/massa-web3.rst:27]: tic-tac-toe will not work for testnet 16

[the remark this answers is not in the document]
Comment (AurelienFT) [reply web3-dev/massa-web3.rst:27]: Ok so I don't think it's a good idea to link unmaintained examples

File: web3-dev/useful-resources.rst (hunk 2 above)
Comment (qdrn) [reply web3-dev/useful-resources.rst:15]: Not sure what you refer to ?

File: web3-dev/massa-web3.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (qdrn) [reply web3-dev/massa-web3.rst]: blog is maintained. We should update some examples with the new tooling and make sure it works as it's useful for newcomers. We don't have to update everything tho'.

**Review by Trottoir — APPROVED** [review]
Ok for me

File: web3-dev/useful-resources.rst (hunk 2 above)
Comment (AurelienFT) [reply web3-dev/useful-resources.rst:15]: The line explain `massa-sc-examples` but have a link to `massa-web3` @qdrn

Comment (qdrn) [reply web3-dev/useful-resources.rst]: Thanks. Did not notice it was on another file. Corrected.

**Review by AurelienFT — APPROVED** [review]

**Pull request merged**